packages/panna-sdk/src/react/components/ui/form.tsx

```diff
@@ -0,0 +1,167 @@
+'use client';
```

Comment:
this is a only nextjs statement. Since the panna-sdk is framework agnostic, not sure if we should have it. The same comment applies for other files that have the same statement

Reply:
Done.